06_gpu/import_torch.py

```diff
@@ -0,0 +1,31 @@
+# # PyTorch with CUDA GPU support
+#
+# This example shows how you can use CUDA GPUs in Modal, with a minimal PyTorch
+# image. You can specify GPU requirements in the `stub.function` decorator.
+
+import time
+
+import modal
+
+stub = modal.Stub(
+    "example-import-torch",
+    image=modal.Image.debian_slim().pip_install(["torch"], "https://download.pytorch.org/whl/cu116"),
+)
+
+
+@stub.function(gpu=True)
+def gpu_function():
+    import subprocess
+    import torch
+
+    subprocess.run(["nvidia-smi"])
+    print("Torch version:", torch.__version__)
+    print("CUDA available:", torch.cuda.is_available())
+    print("CUDA device count:", torch.cuda.device_count())
+
+
+if __name__ == "__main__":
+    t0 = time.time()
+    with stub.run():
+        gpu_function()
+    print("Full time spent:", time.time() - t0)
```

Comment:
Analogous to `import_sklearn.py`, I'm printing out the full time spent on the client wall clock.

<img width="584" alt="image" src="https://user-images.githubusercontent.com/7550632/204149886-6f85eb00-3bae-40ee-881a-6960f8bdf073.png">